src/components/map/MapContainer.jsx

```
@@ -9,13 +9,16 @@ import useLoadingStore from '../../stores/common/useLoadingStore.js';
 import LoadingSpinner from '../common/LoadingSpinner.jsx';
 import usePolylineStore from '../../stores/map/usePolylineStore.js';
 
+const idleCallback = window.requestIdleCallback || ((cb) => setTimeout(cb, 0));
+const cancelIdle = window.cancelIdleCallback || ((id) => clearTimeout(id));
```

ESLint

[error] 'cancelIdle' is assigned a value but never used.

(no-unused-vars)